src/components/include/connection_handler/connection_handler.h

```diff
@@ -325,6 +325,13 @@ class ConnectionHandler {
    */
   virtual void CreateWebEngineDevice() = 0;
 
+  /**
+   * @brief Checks if such an application is in any of the connections.
+   * @param connection_key Application's connection key.
+   * @return True if the application present in the any connections.
+   **/
+  virtual bool IsAppPresentInConnections(const uint32_t connection_key) = 0;
+
```

Comment:
@ydementieiev 
Maybe ? :
virtual bool IsAppPresentInConnections(const uint32_t connection_key) const = 0;

Reply:
@SNiukalov agree, I changed to const [here](https://github.com/LuxoftSDL/sdl_core/pull/44/commits/13eb2ff321b0840753f7159afe7c01806405211a)